boggs/src/main/java/com/hartwig/healthchecks/boggs/extractor/BoggsExtractor.java

```diff
@@ -1,35 +1,48 @@
 package com.hartwig.healthchecks.boggs.extractor;
 
-import java.io.FileNotFoundException;
+import static java.util.stream.Collectors.toCollection;
+import static java.util.stream.Collectors.toList;
+
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.Map;
-import java.util.Optional;
-import java.util.stream.Collectors;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jetbrains.annotations.NotNull;
 
 public class BoggsExtractor {
 	protected static final String FILENAME = "Filename";
 	protected static final String SEPERATOR_REGEX = "\t";
 	protected static final String TOTAL_SEQUENCES = "Total Sequences";
 	protected static final String MAPPING = "mapping";
 	protected static final String FASTQC_DATA_FILE_NAME = "fastqc_data.txt";
+	protected static final String FILE_NOT_FOUND = "File %s was not found";
+	private static final Logger LOGGER = LogManager.getLogger(BoggsExtractor.class);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['member def modifier' have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/hmf/health-checks/file/2780681836/issues/source?bid=3363458&fileBranchId=3365368#l26)